Automattic/wp-calypso
# Google Apps: Show Current User

Fixes #603.

Requires D1179-code

This changes enables the users to see the current list of Google Apps users per domain and per site.

There quite some changes in this PR, namely I moved the Google Apps store to Redux. Since this component also needs non-Redux data as well, they are glued in the data component. 

### Screenshots

#### Sitewide - Has Google Apps
![image](https://cloud.githubusercontent.com/assets/991022/13342445/5b3b544c-dbf6-11e5-8ca5-750adabd77fb.png)

#### Per domain - Has Google Apps
![image](https://cloud.githubusercontent.com/assets/991022/13342458/98c8013e-dbf6-11e5-99e5-28b3eab84b68.png)

#### Sitewide - No domains
![image](https://cloud.githubusercontent.com/assets/991022/13342479/d70ceb8a-dbf6-11e5-8b29-26dfe16a85d1.png)

#### Sitewide - Only mapped domains
![image](https://cloud.githubusercontent.com/assets/991022/13557300/9687e0fc-e3a2-11e5-9655-01139f33174a.png)


#### Per domain - Not eligible
![image](https://cloud.githubusercontent.com/assets/991022/13557302/9e0d8714-e3a2-11e5-83ed-1a42e5a48c6e.png)

### Testing
1. Go to domain management -> Email 
2. Ensure they look like the screenshots above and the links work.
3. Ensure Add button is hidden if you're not an admin

1. Go to domain management -> Select domain -> Click Email
2. Ensure they look like the screenshots above and the links work

/cc: @klimeryk @stephanethomas @mtias for code review (first attempt at Redux – extra eyes welcome)
/cc: @breezyskies for design
/cc: @ranh for copy
